launch/tower_building.launch

```diff
@@ -5,15 +5,19 @@
 
     <arg name="forceThresholdL"      default="2.75"/>    <!-- [N] -->
     <arg name="forceThresholdR"      default="2.0"/>    <!-- [N] -->
+    <arg name="forceFiltVarianceL"   default="0.001"/>
+    <arg name="relForceThresholdL"   default="65.0"/>
     <arg name="speechPopUpDuration"  default="20.0"/>    <!-- [s] -->
 
     <!-- If the robot will recover from an error or will wait the external planner's decisions -->
-    <arg name="internalRecovery"    default="true"/>
+    <arg name="internalRecovery"    default="false"/>
     <!-- If the system should use the robot or not (useful for debugging) -->
     <arg name="useRobot"            default="true"/>
 
     <param name="action_provider/force_threshold_left"  value="$(arg forceThresholdL)"/>
     <param name="action_provider/force_threshold_right" value="$(arg forceThresholdR)"/>
+    <param name="action_provider/force_filt_variance_left" value="$(arg forceFiltVarianceL)"/>
+    <param name="action_provider/rel_force_threshold_left" value="$(arg relForceThresholdL)"/>
```

Comment:
Where is the right arm? Also, for `param` variables you should use longer (more clear to understand) names, e.g. `relative_force_threshold_left`